.evergreen/config.yml

```diff
@@ -1035,43 +1035,47 @@ axes:
       - id: "latest"
         display_name: "MongoDB latest"
         variables:
-           VERSION: "latest"
+           MONGODB_VERSION: "latest"
+      - id: "rapid"
+        display_name: "MongoDB rapid"
+        variables:
+           MONGODB_VERSION: "rapid"
       - id: "6.0"
         display_name: "MongoDB 6.0"
         variables:
-           VERSION: "6.0"
+           MONGODB_VERSION: "6.0"
       - id: "5.0"
         display_name: "MongoDB 5.0"
         variables:
-           VERSION: "5.0"
+           MONGODB_VERSION: "5.0"
       - id: "4.4"
         display_name: "MongoDB 4.4"
         variables:
-           VERSION: "4.4"
+           MONGODB_VERSION: "4.4"
       - id: "4.2"
         display_name: "MongoDB 4.2"
         variables:
-           VERSION: "4.2"
+           MONGODB_VERSION: "4.2"
       - id: "4.0"
         display_name: "MongoDB 4.0"
         variables:
-           VERSION: "4.0"
+           MONGODB_VERSION: "4.0"
       - id: "3.6"
         display_name: "MongoDB 3.6"
         variables:
-           VERSION: "3.6"
+           MONGODB_VERSION: "3.6"
 
   - id: mongodb-edge-versions
     display_name: MongoDB Version
     values:
       - id: "latest-stable"
-        display_name: "MongoDB 5.0"
+        display_name: "MongoDB 6.0"
         variables:
-          VERSION: "5.0"
+          MONGODB_VERSION: "6.0"
```

Comment:
Thoughts on using "rapid" here?

See also: https://github.com/mongodb/mongo-php-library/pull/1056#discussion_r1158056586

Reply:
SGTM 👍

Reply:
Quoting @JamesKovacs from Slack:

> When M.0.0 goes GA, rapid points to (M-1).3.x for a quarter, which isn't useful since no one can run that quarterly release

With that in mind, I think keeping 6.0 makes sense here and we can just bump to 7.0 when that gets released.